lib/Fencer/Rules.hs

```diff
@@ -26,22 +26,28 @@ import Fencer.Types
 --
 -- Throws an exception for unparseable or unreadable files.
```

Comment:
Ok, so this skips all dot-directories including `subDirectory` and below. Right? If yes, can you add it to the haddock?

Reply:
That's right. I expanded the documentation for this.